### ISSUE TITLE
[FEATURE]: Add twitter to contributors from api

### ISSUE THREAD

**Issue by MohamedAlDeep** [devvsakib/github-error-solve#104]
**Is your feature request related to a problem? Please describe.**
Yes, it is related to #103 it is all described there

**Describe the solution you'd like**
Described in #103 
**Describe alternatives you've considered**
N/A
**Additional context**
N/A

**Comment by devvsakib** [comment]
yeah I'm currently working on it

**Comment by MohamedAlDeep** [comment]
@devvsakib I am interested in this and i am learning about it so i am also doing it, i wanted to ask you about this
![image](https://user-images.githubusercontent.com/95958815/214629863-6e5dc1c5-68fd-4b77-94b0-394316c9762d.png)
i reached to this but i dont know how to fetch the data from the new URLs
I dont know if you did this and i dont know how you are going to do it , if you did it then i will check how you did it, if you didnt then lets help each other because api things are kinda new to me

**Comment by MohamedAlDeep** [comment]
the line breaks the two functions (visually)
```js
import { useState, useEffect } from 'react'

function App() {
// ================================================================== 
  const url = 'https://api.github.com/repos/devvsakib/github-error-solve/contributors'
  const [data, setData] = useState([])
  useEffect( () => {
    const fetchUsers = async (url) => {
      const response = await fetch(url)
      const users = await response.json()
      setData(users)
    }
    fetchUsers(url)
  }, [data.length])
// =================================================================== 
const [userData, setuserData] = useState([])
var userUrl
data.map( data => {
      userUrl = `https://api.github.com/users/${data.login}`
    console.log(userUrl)

   

  }
  
  )
  useEffect( () => { 
    const fetchTwitter = async (userUrl) => {
      const Userresponse = await fetch(userUrl)
      const twitter = await Userresponse.json()
      setuserData(twitter)
    }
    fetchTwitter(userUrl)
    }, [userData.length])
  
// ===================================================================
  return (
    
    <div> 
    
      {
        userData.map(userData => 
          <h1>{userData.twitter_username}</h1>
          )
      }
    </div>
  )
}

export default App
```

**Comment by MohamedAlDeep** [comment]
actually i got another idea

**Comment by MohamedAlDeep** [comment]
nvm all that code

**Comment by MohamedAlDeep** [comment]
God i just got the same product with less code LMAO
### Image
![image](https://user-images.githubusercontent.com/95958815/214636195-f5e2ed8c-b930-433e-b93a-c68f0c72112c.png)
### Code
```js
import { useState, useEffect } from 'react'

function App() {
// ================================================================== 
  const url = 'https://api.github.com/repos/devvsakib/github-error-solve/contributors'
  const [data, setData] = useState([])
  useEffect( () => {
    const fetchUsers = async (url) => {
      const response = await fetch(url)
      const users = await response.json()
      setData(users)
    }
    fetchUsers(url)
  }, [data.length])
// =================================================================== 

// ===================================================================
  return (
    
    <div> 
    
      {
       data.map(data => 
          <h1>{data.url}</h1>
          )
      }
    </div>
  )
}

export default App

```

**Comment by MohamedAlDeep** [comment]
My problem is in fetching that url

**Comment by MohamedAlDeep** [comment]
@devvsakib I JUST FOUND THE SOLUTION

**Comment by MohamedAlDeep** [comment]
![image](https://user-images.githubusercontent.com/95958815/214649632-d0dc73ca-d731-4981-8e71-1293ecc1ec9e.png)

**Comment by MohamedAlDeep** [comment]
for a reason i dont know it keeps repeating

**Comment by MohamedAlDeep** [comment]
bruh its not working now

**Comment by devvsakib** [comment]
Im in nutshell with these API's😂 I wrote some code but didn't test it. I have to do an assignment too😂

I will suggest you try all this API fetch in vanilla javascript. first, fetch contributors then again users.👀

**Comment by MohamedAlDeep** [comment]
Yes, that is good idea but for now i cant because it says "rate limiting" when opening the contributors API 😂

**Comment by devvsakib** [comment]
yeah  GitHub API has limitation for request

**Comment by devvsakib** [comment]
Closed as complete #115
